### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by jweir211**
for the page template

### CODE REVIEW

**Review by Arcfej — CHANGES_REQUESTED** [review]
I reviewed only the index.html and master.css, as that's what I think you intended as a template. I added a requested change and some suggestions. Due to the short amount of time, you may ignore the suggestions, but I think they would improve teamwork. I'd also like to ask, and Idris also mentioned this on Teams, to convert the html files to php, so that we can easily use them on the backend.

File: Pages/styling/master.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.btn{
+  margin-right: 10px;
+  color: white;
+}
+
+.list-group{
+  max-height: 350px;
+}
```
[the remark this answers is not in the document]
Comment (jweir211) [reply Pages/styling/master.css:29]: this is not for a colour? It is to stop tghe list of rules running down off the page and allow for the scroller

File: Pages/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+  <link rel="stylesheet" href="styling/master.css">
+
+  <title>This is the title of the webpage!</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-lg fixed-top navbar-dark navbar-expand border" style="background-color: #115E67;">
```
[the remark this answers is not in the document]
Comment (jweir211) [reply Pages/index.html:16]: Must leave as is. bootstrap css overwrites my css for bg-primary

**Comment by Arcfej** [comment]
Changes have been merged by another PR

**Pull request closed without merging**